### DISCUSSION TITLE
大家覺得到底誰才是GOAT LBJ V.S. MJ

### DISCUSSION THREAD

**Discussion by Teddy111802518** [1101-GS4719-C/Python-Community-GS4719-C#2110] [General]
一直以來到底誰才是GOAT是大家熱愛討論的話題，不同時代有不一樣的引領聯盟的人物，但最常拿來比較的就是Michael Jordan和Lebron James 歡迎大家提供自己的想法 說不定你覺得林書豪才是GOAT!!!

**Comment by Teddy111802518** [comment]
https://www.sportsv.net/articles/114170
https://www.sportsv.net/articles/114238

**Comment by Teddy111802518** [comment]
啊我個人是支持LBJ啦 因為我沒看過MJ打球 他然很厲害 榮譽滿滿 但我認為籃球是團體運動  所以我更喜歡技術全面並且開發對有潛力的球員

**Reply by Karsta0216** [reply]
https://new.qq.com/rain/a/20210218A0373S00
GOAT的原文是: Greatest Of All Time。我的想法和文章中提及的一樣，我認為拿不同時期的球員互相比較是不可行的。因此，我並沒有覺得誰是GOAT。但我們可以說每個時期都有某些或某位球員特別突出，他就像火炬一樣一代傳著一代。張伯倫、奧拉朱萬、伯德、艾弗森、魔術強森、喬丹、歐尼爾、柯比、鄧肯、詹姆斯、諾維斯基、杜蘭特、柯瑞、唐西奇、約基奇、塔圖姆，有太多太多球員在各自的時期發揚光大。我們無法讓他們在彼此巔峰的時候同台較量，也因為時代的不同，數據的比較有時會失去其公平性，也因為每一位球迷熱愛的球風都不一樣，導致我們很難去說誰才是GOAT。就像自古以來在科學界，誰是你心中的GOAT？愛因斯坦？牛頓？

**Reply by Teddy111802518** [reply]
我認為你說的沒錯， 確實不同年代很難去比較，但他也確實是一個長於飯後的好話題，我也只能在遊戲2K上進行對決來看誰帶領的隊伍最強，至於科學界我不知道誰是goat，但我一定會成為震驚歷史的人物，如果我完成我理想的開發的話
